content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/create-github-issues/index.md

```diff
@@ -0,0 +1,235 @@
+      + name                       = "enforce-events-issue-opener"
+      + project                    = "<google project>"
+      + status                     = (known after apply)
+. . .
+To perform exactly these actions, run the following command to apply:
+    terraform apply "enforce-events.plan"
+```
+
+Inspect the plan to ensure that the resources that Terraform will create are in line with your Google Cloud project's settings and IAM permissions. Once you are satisfied with the plan, apply it:
+
+```shell
+terraform apply enforce-events.plan
+```
+
+Terraform will run the module and you will receive output like the following:
+
+```
+google_secret_manager_secret.gh-pat: Creating...
+google_service_account.gh-iss-opener: Creating...
+. . .
+Outputs:
+
+secret-command = "echo -n YOUR GITHUB PAT | gcloud --project <google project> secrets versions add enforce-events-github-pat --data-file=-"
+url = "https://enforce-events-issue-opener-ntfj7ezpja-uc.a.run.app"
+```
+
+Note down the `url`, and the secret-command.
+
+## Configuring Google Cloud with your Github Token
+
+With the output from the `terraform apply` step, you can now run the `echo` command to configure Google Cloud with your Github personal access token.
+
+Substitute your Github personal access token in place of the `YOUR GITHUB PAT` string and run the following command:
+
+```shell
+echo -n <your github PAT> | gcloud --project <google project> secrets versions add enforce-events-github-pat --data-file=-
+```
+
+This configures your Google Cloud project with your token, so that it is available to the demo application when it is invoked on Cloud Run.
+
+### Subscribing to Enforce Cloudevents
+
+Now that the demo application is deployed, the next step is to register it as a Cloudevents receiver using Enforce.
+
+Run the following `chainctl` command to create an Enforce webhook subscription:
+
+```shell
+chainctl events subscriptions create \
+  --group <your chainguard group> \
+  https://<url from terraform>
+```
+
+You will receive output showing the ID of the subscription, and the webhook endpoint like the following:
+
+```
+                    ID                   |             SINK
+ ----------------------------------------+-----------------------------
+  <chainguard group ID>/0dfbc3c8b89e0147 | https://<url from terraform>
+  ```
+
+Enforce will now send events to the application that is running on Cloud Run at the specified URL.
+
+## Configuring an Enforce Policy
+
+Before you can open Github issues for policy violations, you will need to ensure that you have Enforce configured with a policy that you can test. Make sure you have a cluster registered with Enforce using the `chainctl clusters ls` command. If you do not, visit our [Getting Started with Enforce Guide](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding/#step-3--prepare-kubernetes-cluster) to create and register a cluster.
```

Comment:
Be sure to make all the URLs relative - and to update them post-migration

Reply:
Ah migration is done, will update before we merge this